vars/go.groovy

```diff
@@ -8,10 +8,12 @@ def ccv() {
         cd temp
         git clone https://${GITHUB_CREDS_PSW}@github.com/helxplatform/${REPO_NAME}.git > /dev/null
         cd ${REPO_NAME}
-        git switch ${BRANCH_NAME} > /dev/null
-        if [ $? != 0 ]  ; then
-            echo "ccv(): ERROR: Unable to switch to branch ${BRANCH_NAME} to set ccv, exiting.";
-            exit;
+        if [ ${BRANCH_NAME} = "master -o $BRANCH_NAME = "main" ]; then
```

Comment:
missing a `"` after `"master` I think

Reply:
also I would use `${BRANCH_NAME}` everywhere instead of `$BRANCH_NAME` just in case

Reply:
also I would use `$BRANCH_NAME` within the single brackets. I'm not sure if ${} will work between those, they're finicky.